### DISCUSSION TITLE
Contributor notes

### DISCUSSION THREAD

**Discussion by GrumpyMeow** [tteck/Proxmox#2702] [General]
How to generate the ASCII art?
Could you maybe provide the service you use?

I've now used: https://patorjk.com/software/taag/#p=display&f=Slant&t=Frigate

**Comment by tteck** [comment]
That's the one 👍🏻

**Comment by GrumpyMeow** [comment]
Is there any trick how i can run my script on my Proxmox host to see it in action?
Eventually it should use: `bash -c "$(wget -qLO - https://github.com/tteck/Proxmox/raw/main/ct/frigate.sh)"`.
But during development.. Should i use VSCode-remote-SSH extension for instance?

**Reply by GrumpyMeow** [reply]
Aha got it:
`bash -c "$(cat ct/frigate.sh)"`
Doesnt work. As the scripts pull directly from the main GitHub-repos..
Gonna try it another way.
